src/main/java/seedu/address/model/person/PersonFactory.java

```diff
@@ -74,21 +43,24 @@ public static Person createPerson(Role role, Name name, Phone phone, Email email
         switch (role) {
         case VOLUNTEER:
             if (argMultimap.getValue(PREFIX_HOURS).isEmpty()) {
-                throw new ParseException("Hours must be specified for a Volunteer.");
+                throw new ParseException("Missing hours! "
+                        + "For a Volunteer, you must specify hours contributed by the volunteer using the h/ prefix.");
```

Comment:
I am thinking we can extract the checks for each case out into its own methods to make things cleaner? Let me know what you think! Other than that looks good :D

Reply:
That's a good idea! Let me do it tomorrow